README.md

```diff
@@ -29,15 +29,23 @@ Machine, which is required to run Docker on Mac/Windows hosts.
 After you clone the repository locally, there are several configuration steps required to prepare the project.
 
 
-#### Create settings_secret.py, settings_local.py, and secrets.py
+#### Create settings and secrets
+
+##### Portal
 
 - Create `server/portal/settings/settings_secret.py` containing what is in `secret` field in the `Core Portal Settings Secret` entry secured on [UT Stache](https://stache.utexas.edu)
 
 - Copy `server/portal/settings/settings_local.example.py` to `server/portal/settings/settings_local.py`
-    - _Note: [Setup ngrok](#setting-up-notifications-locally) and update `WH_BASE_URL` in `settings_local.py` to enable webhook notifications locally._
+    - _Note: [Setup ngrok](#setting-up-notifications-locally) and update `WH_BASE_URL` in `settings_local.py` to enable webhook notifications locally_
+
+##### CMS
 
 - Copy `server/conf/cms/secrets.sample.py` to `server/conf/cms/secrets.py`
```

Comment:
We'll need dummy `server/conf/cms/settings_local.py` and `server/conf/cms/settings_custom.py` for their use in the docker-compose.

Maybe this could be changed to (for easy copy/paste)

```
cp server/conf/cms/secrets.sample.py server/conf/cms/secrets.py
touch server/conf/cms/settings_local.py
touch server/conf/cms/settings_custom.py
```